INSTALL.md

```diff
@@ -118,15 +126,15 @@ be edited here.
         $ NODE_ENV="development" 
 
         ### Before you run this command, update the config/config.json with the recently created database username and password.
-        ### 
-        ### This will spew a lot of information to the screen and may
-        ### take several minutes.  It creates the tables and loads data
-        ### into UsAddress and SelectedCounties.
-        $ node data/import_into_postgres.js
 
+        ### Run sequelize migrations
+        $ sequelize db:migrate
 
 1. Load sample data and run migrations
 
+        ### It creates the tables and loads data into UsAddress and SelectedCounties.
+        $ node data/import_into_postgres.js
```

Comment:
But then this sample data has to be *always* kept up-to-date with respect to the latest DB schema, right?  IOW, we do an initial schema setup, which gets us to state X, followed by a schema migration(s), which gets us to state Y, and *then* we load sample data that will always depend on the DB already being in state Y.

Have I understood correctly?  And, if I have, I think it would be good for the comments to explain this.

Reply:
Can update the comments, and yes, you're correct.  Development data and migrations are always kind of at odds.